lib/autobuild/packages/python.rb

```diff
@@ -0,0 +1,79 @@
+require 'autobuild/configurable'
+
+# Main Autobuild module
+module Autobuild
+    def self.python(opts, &proc)
+        Python.new(opts, &proc)
+    end
+
+    # Handler class to build python-based packages
+    class Python < Configurable
+        attr_accessor   :buildflags
+        attr_accessor   :installflags
+
+        @builddir = 'build'
```

Comment:
This does ... nothing

Reply:
Yeah.. I was wondering why you did that on both cmake and autotools package handlers. Will remove :+1: